### PR TITLE
DB-6274 Reduce logging on MemStoreAwareScanner

### PR DESCRIPTION
**Pull request by dgomezferro**
There is too much logging and that's creating issues on Jenkins

### CODE REVIEW

File: hbase_storage/hbase1.1.0/src/main/java/com/splicemachine/access/client/MemStoreFlushAwareScanner.java [hunk 1]
```diff
@@ -83,13 +83,16 @@ public KeyValue peek() {
                 return ClientRegionConstants.MEMSTORE_BEGIN_FLUSH;
             }
         }
-        if (beginRow)
+        if (beginRow) {
+            if (LOG.isTraceEnabled())
+                SpliceLogUtils.trace(LOG, "Peek: memstore begin");
```
Comment (yxia92) [inline hbase_storage/hbase1.1.0/src/main/java/com/splicemachine/access/client/MemStoreFlushAwareScanner.java]: Why this change is not applied to 1.0.0.x and 1.0.0?

File: hbase_storage/hbase1.1.0/src/main/java/com/splicemachine/access/client/MemStoreFlushAwareScanner.java [hunk 2]
```diff
@@ -119,19 +122,27 @@ public boolean reseek(KeyValue kv) throws IOException {
     public boolean internalNext(List<Cell> outResult,ScannerContext scannerContext) throws IOException {
         if (beginRow) {
             beginRow = false;
+            if (LOG.isTraceEnabled())
+                SpliceLogUtils.trace(LOG, "Next: memstore begin");
```
Comment (yxia92) [inline hbase_storage/hbase1.1.0/src/main/java/com/splicemachine/access/client/MemStoreFlushAwareScanner.java]: Why this change is not applied to 1.0.0.x and 1.0.0?

File: hbase_storage/hbase1.1.0/src/main/java/com/splicemachine/access/client/MemStoreFlushAwareScanner.java [hunk 3]
```diff
@@ -140,6 +151,8 @@ public boolean internalNext(List<Cell> outResult,ScannerContext scannerContext)
                 }
             } else {
                 flushAlreadyReturned = true;
+                if (LOG.isTraceEnabled())
+                    SpliceLogUtils.trace(LOG, "Next: returning begin flush ");
```
Comment (yxia92) [inline hbase_storage/hbase1.1.0/src/main/java/com/splicemachine/access/client/MemStoreFlushAwareScanner.java]: Why this change is not applied to 1.0.0.x and 1.0.0?

Comment (dgomezferro) [reply hbase_storage/hbase1.1.0/src/main/java/com/splicemachine/access/client/MemStoreFlushAwareScanner.java]: They weren't too important and the commits will have to be reverted or cleaned up after debugging the wrong result in jenkins, but I cleaned up this piece and added the missing logs. Thanks for the review!
